addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/handler/AutelisHandler.java

```diff
@@ -112,18 +135,18 @@
     private HttpClient client = new HttpClient();
 
     /**
+     * Authentication for login
+     */
+    String basicAuthentication;
```

Comment:
Can this be private as well, maybe with some getter (and / or setter)

Reply:
Yep